### PR TITLE
#2176 - Make long relations nicer without arcs

### PR DESCRIPTION
**Pull request by jcklie**
**What's in the PR**
- Option to not render relations

Highlighting of the other span when hovering/rendering of this via equivalenve sets is still missing.

**How to test manually**
* ...

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

**Todo**
- [ ] Make adding equivalence sets fast
- [ ] Add hover for same equivalence sets in BRAT

### CODE REVIEW

File: inception/inception-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/adapter/SpanAdapter.java [hunk 1]
```diff
@@ -77,6 +81,13 @@ public SpanAdapter(LayerSupportRegistry aLayerSupportRegistry,
         }
     }
 
+    @Override
+    public void initialize(AnnotationSchemaService aSchemaService)
```
Comment (reckart) [inline inception/inception-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/adapter/SpanAdapter.java:85]: This method is called in order to create default features in the annotation schema when a new layer is being created. This is *not* meant to be used to initialize the adapter itself. It is called only by `LayerDetailForm.actionSave`. You cannot reliably obtain an `annotationSchemaService` in this way.

**Comment by jcklie** [comment]
I do not like the name of that strategy but I do not know a better name for it.

**Comment by reckart** [comment]
Looking repeatedly for attached relations via the method offered by the annotation service is quite slow - not something I think is a good idea to do during rendering. It is ok in the annotation sidebar where we call it only once, but in a loop as during rendering it doesn't scale well. Searching the `VDocument` would probably be quite a bit faster.

**Comment by jcklie** [comment]
> Looking repeatedly for attached relations via the method offered by the annotation service is quite slow - not something I think is a good idea to do during rendering. It is ok in the annotation sidebar where we call it only once, but in a loop as during rendering it doesn't scale well. Searching the `VDocument` would probably be quite a bit faster.

I do not know how to get the relations from the vdoc, as the spans are rendered first and then the relations.

**Pull request closed without merging**